apps/frontend/src/app/5_pages/LeaderboardPointsPage/hooks/useGetPoints.ts

```diff
@@ -1,52 +1,80 @@
-import { useMemo } from 'react';
+import { useCallback, useMemo } from 'react';
 
 import { useAccount } from '../../../../hooks/useAccount';
 import { areAddressesEqual } from '../../../../utils/helpers';
 import {
   EXTRA_SPICE_POINTS_MULTIPLIER,
   RUNES_POINTS_MULTIPLIER,
 } from '../LeaderboardPointsPage.constants';
-import { User, UserPoints } from '../LeaderboardPointsPage.types';
+import { UserPoints } from '../LeaderboardPointsPage.types';
+import usersExtraPointsList from '../data/usersExtraPoints.json';
 import usersPointsList from '../data/usersPoints.json';
 
 const data: UserPoints[] = usersPointsList;
+const extraData = usersExtraPointsList.map(({ toAddress, points }) => ({
+  wallet: toAddress,
+  extraSpiceShot: parseFloat(points),
+}));
 
 export const useGetPoints = () => {
   const { account } = useAccount();
 
+  const mergeExtraPoints = useCallback(
```

Comment:
Move it to `LeaderboardPointsPage.utils.tsx`. I would also adjust it so it already returns sorted data.